pkg/apis/kubeone/v1beta2/types.go

```diff
@@ -95,6 +99,14 @@ type KubeOneCluster struct {
 	ControlPlaneComponents *ControlPlaneComponents `json:"controlPlaneComponents,omitempty"`
 }
 
+type CAConfig struct {
+	// CABundle inline PEM encoded global CA
+	CABundle string `json:"caBundle,omitempty"`
+
+	// CAFile is a path to the CA bundle file, used as a replacement for Bundle
+	CAFile string `json:"caFile,omitempty"`
+}
```

Comment:
- We could also rename the type to `CertificateAuthorithyConfig` to be consistent with other names
- I think `CABundle` and `CAFile` are redundant, what do you think about just having `bundle` and `file`?

Reply:
Yeah, after rename to CertificateAuthorithy renaming CABundle to Bundle reads good